hassio/secrets.py

```diff
@@ -18,7 +18,7 @@ class SecretsManager(CoreSysAttributes):
     def __init__(self, coresys: CoreSys):
         """Initialize secret manager."""
         self.coresys: CoreSys = coresys
-        self.secrets: Dict[str, Union[bool, float, int, str]] = {}
+        self.secrets: Dict[str, Union[bool, float, int, str, List[int]] = {}
```

Comment:
Why is this only a list of integers?

Reply:
No reason whatsoever, other than that’s all that my add-on’s specific use case would require. I’m happy to expand it to whatever you all deem acceptable.